website/templates/project/register.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="title()">Registration of ${node['title']}</%def>
   <div id="registrationMetaDataScope" class="container scripted">
     <div class="row">
-      <div class="span8 col-md-2 columns eight large-8">        
+      <div class="span8 col-md-2 columns eight large-8">
```

Comment:
It seems like you identified some specific classnames here that can be removed. It would save me time if you could indicated which ones explicitly.